src/_help/changes-management/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changes management
+---
+
+- TOC
+{:toc}
+
+Tracking API changes is essential, both for users and developers.
+
+## What is a change
+
+Changes are differentiated into various categories:
+
+| Documentation change     | a description is modified  |
+| Structural change        | the structure of the API definition is modified  |
+| Breaking change    | a major change in your API definition requires an update to the code consuming the API for its users |
+
+Breaking changes are the ones your API consumers should not miss and are highlighted in the changelog. Here is a non-exhaustive list of criteria that help us determine if a change is breaking or not:
+- Renaming/Deleting an endpoint or operation
+- Renaming/Deleting a property (body, parameters, etc...)
+- Changing the "type" attribute of a property
+- Removing polymorphism of a property
+- Marking an existing property as required
+- Add or delete a security requirement
```

Comment:
This should be an exhaustive list. Also, WDYT of moving this to the `Concepts` page? I think it would be the best place to define what's a breaking change for us. We should also add `API structure change` concept, and `Documentation change`.

Reply:
Will stick to Breaking changes today. Will work on the last ones later.
I haven't been able to find any exhaustive list, anywhere. So I'd try to investigate further more or even request help from the team.

Reply:
:+1: